app/router.js

```diff
@@ -22,6 +22,9 @@ Router.map(function() {
       this.route('new');
     });
     this.route('dedupe', function() {});
+    this.route('accounts', function() {
+      this.route('list', {path: '/'});
```

Comment:
A couple things to note here.
* if you leave out `this.route('list', {path: '/'});`, then by default, it will be as if `this.route('index', {path: '/'});` was put there.  So by convention, the recommendation is to go with the default `index` route.
* In this particular case, I think you can work with the `accounts` route and not nest anything inside.  If in the future we decide that we need to further next routes, we can then.  So my recommendation is to do this:
```
this.route('accounts');
```

...and then just work with the `accounts/route.js` and `accounts/template.hbs`

I also noticed that `this.route('dedupe', function() {});` should probably be replaced with `this.route('dedupe');` (a mistake that I made :-)).